src/PrestaShopBundle/Resources/config/services/bundle/translation.yml

```diff
@@ -57,7 +57,7 @@ services:
             - '@prestashop.translation.extractor.theme'
             - '@prestashop.core.addon.theme.repository'
             - '@filesystem'
-            - '%themes_translations_dir%'
```

Comment:
@atomiix Is there a reason why you fixed here, but you didn't fix also on line 108 https://github.com/PrestaShop/PrestaShop/blob/6e4a76d183753fcef267eb565710600a439dc8d2/src/PrestaShopBundle/Resources/config/services/bundle/translation.yml#L108

The `PrestaShopBundle\Translation\Provider\ThemeProvider` doesn't find the theme translation neither.